packages/storage-evm/scripts/base-mainnet-contracts-scripts/ENERGY_DISTRIBUTION_API.md

```
@@ -0,0 +1,355 @@
+function parseUSDC(dollarAmount) {
+  return ethers.parseUnits(dollarAmount.toString(), 6);
+}
+```
+
+---
+
+## 📍 Contract & Member Addresses
+
+### Contract Address (Base Mainnet)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)